utils/graph.c

```diff
@@ -170,9 +188,14 @@ static int add_graph_exit(struct uftrace_task_graph *tg)
 	}
 
 out:
-	node->time += fstack->total_time;
+	node->total_time.sum += fstack->total_time;
 	node->child_time += fstack->child_time;
 
+	update_graph_time_stat(&node->total_time, fstack->total_time);
+	if (node->nr_calls > 0) {
+		node->total_time.avg = (node->total_time.sum) / node->nr_calls;
```

Comment:
This also can be done in the `update_graph_time_stat` but I think we can just drop 'avg' from the stat as it can be easily calculated from sum and calls when it's printed.